src/libsyntax_ext/deriving/clone.rs

```diff
@@ -80,6 +80,7 @@ pub fn expand_deriving_clone(cx: &mut ExtCtxt,
         additional_bounds: bounds,
         generics: LifetimeBounds::empty(),
         is_unsafe: false,
+        supports_unions: false,
```

Comment:
ideally, we would support `#[derive(Copy, Clone)]` but not `#[derive(Clone)]` in isolation